README.md

```diff
@@ -88,14 +88,15 @@ Before starting this codelab, make sure that you've installed:
     1. Open your app at http://localhost:5000
         You should see your copy of FirebaseRTC which has been connected to your Firebase project.
     The app has automatically connected to your Firebase project.
+    > Note: if you want to access the server from other devices on your local network, you need to add `-o 0.0.0.0` at the end of the `firebase serve` command.
     
 1. __Creating a new room__
     
     In this application, each video chat session is called a __room__. A user can create a new room by clicking a button in their application. This will generate an ID that the remote party can use to join the same room. The ID is used as the key in Cloud Firestore for each room.
     
     Each room will contain the `RTCSessionDescriptions` for both the offer and the answer, as well as two separate collections with [ICE candidates](https://webrtcglossary.com/ice/#:~:text=ICE%20stands%20for%20Interactive%20Connectivity,NAT%20traversal%20used%20in%20WebRTC.) from each party.
 
-    Your first task is to implement the missing code for creating a new room with the initial offer from the caller. Open public/app.js and find the comment `// Add code for creating a room here` and add the following code:
+    Your first task is to implement the missing code for creating a new room with the initial offer from the caller. Open public/app.js, find the comment `// Add code for creating a room below`, and add the following code. (You should add the code before the corresponding `// Add code for creating a room above`; the same goes for the rest of the code blocks.)
```

Comment:
tbh I didn't really like the syntax of having the double lines, but I like this clarification.  It seemed obnoxious to have to type that multiple times.  Good compromise.